recipe/build.sh

```diff
@@ -32,6 +32,13 @@ else
 fi
 
 if [[ "$CONDA_BUILD_CROSS_COMPILATION" != "1" ]]; then
+   if [[ ${target_platform} =~ .*osx.* ]]; then
+      # hmaarrfk - 2024/08
+      # skip two problematic tests on OSX
+      # https://github.com/conda-forge/gnutls-feedstock/pull/47
+      sed -i.back 's,gnutls-cli-debug.sh,,g' tests/Makefile
+      sed -i.back 's,ocsp-tests/ocsp-must-staple-connection.sh,,g' tests/Makefile
+   fi
```

Comment:
are you ok with skipping the tests.

Reply:
sure ! I'll report the failure upstream but we can go ahead.

Reply:
Let’s add then reference to the issues you will open before merging. 

Thanks!

Reply:
It's there https://gitlab.com/gnutls/gnutls/-/issues/1539

Reply:
ok added it to the script itself. this seems good to me now.